### DISCUSSION TITLE
Cockroach DB not available in the Integrations Page?

### DISCUSSION THREAD

**Discussion by rohitsudhakar-crl** [vercel/community#6161] [Help]
### Summary

Did you remove Cockroach Db from the integrations page?

### Steps to Reproduce

Navigate to Integration Page> Search for Cockroach DB

**Comment by amyegan** [comment]
Thanks for reaching out! The Cockroach Labs team has removed the integration for the marketplace for the moment. But you can still connect manually by adding the appropriate [environment variables](https://vercel.com/docs/projects/environment-variables). 

For more information on how to do that, please check their [connection documentation](https://www.cockroachlabs.com/docs/v23.2/connection-parameters) or reach out to them directly via one of the [Cockroach Labs support channels](https://www.cockroachlabs.com/contact/).
